### PR TITLE
dinput: Add profile for USB PlayStation Adapter

### PR DESCRIPTION
**Pull request by CutterXYZ**

### CODE REVIEW

**Comment by RobLoach** [comment]
Could you provide details about the hardware? Is it a playstation controller?

**Comment by CutterXYZ** [comment]
It is an adapter accepting two PlayStation controllers.

**Comment by CutterXYZ** [comment]
@RobLoach I can provide more info if needed.

File: dinput/USB GamePad .cfg [hunk 1]
```diff
@@ -0,0 +1,55 @@
+input_driver = "dinput"
```
Comment (RobLoach) [inline dinput/USB GamePad .cfg:1]:
``` suggestion
# An adapter accepting two PlayStation controllers
input_driver = "dinput"
```

**Pull request merged**